controllers/OrderController.js

```diff
@@ -0,0 +1,44 @@
+import { OrderService } from '../services';
+
+const findCartItems = async (req, res) => {
+  try {
+    const { id } = req.foundUser[0];
+    const cartItems = await OrderService.findCartItems(id);
+
+    res
+      .status(200)
+      .json({ message: 'FIND_CART_ITEMS_SUCCESSFULLY', cartItems });
+  } catch (err) {
+    res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const updateCartItemQuantity = async (req, res) => {
+  try {
+    const { id } = req.foundUser[0];
+    const { cart_item_id, quantity } = req.body;
```

Comment:
```suggestion
  try {
    const { id } = req.foundUser[0];
    const { cartItemId, quantity } = req.body;
```

SQL 내부에서가 아니라면 모두 카멜케이스로 통일해주세요 :)

Reply:
아앗 ㅠㅠ이전의 리뷰에서도 같은 내용을 지적해주셨는데도 불구하고
또 같은 실수를 해버렸네요.
죄송합니다. SQL 내부가 아니라면 카멜케이스로 사용한다는 점 명심하겠습니다.

https://github.com/wecode-bootcamp-korea/fullstack1-1st-BaseNote-backend/pull/8/commits/21924818365ee86c44b91edd6a0372850c14d694